config/routes.rb

```diff
@@ -1,11 +1,12 @@
 IntergalacticBattleship::Application.routes.draw do
+  devise_for :users
   root "strikes#index"
   resources :strikes, only: [:new, :create]
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'static_pages#index'
```

Comment:
I think this is causing our CI test suite to fail.  Attached is the failure message:

```

1) User add some examples to (or delete) /home/rof/src/github.com/FirehoseCommunity/intergalactic-battleship/spec/models/user_spec.rb
# Not yet implemented
     # ./spec/models/user_spec.rb:4


Failures:

1) StaticPagesController static_pages#index should be displayed to a user
Failure/Error: get :index

ActionController::UrlGenerationError:
  No route matches {:action=>"index", :controller=>"static_pages"}
```